source/life_expectancy_script.R

```diff
@@ -0,0 +1,142 @@
+library(tidyverse)
+library(readxl)

```

Comment:
Would it be better to move all library calls to the main script?

Reply:
Do you want this life_expectancy_script.R to be sourced in the 0-Parameterised_reports.R script?

Reply:
Yes I think so. We're aiming to keep it as simple as possible for whoever runs this in future.